website/project/views/node.py

```diff
@@ -856,7 +856,7 @@ def private_link_table(node, **kwargs):
 @must_be_valid_project
 def get_editable_children(auth, node, **kwargs):
 
-    if not node.can_edit(auth):
+    if not node.has_permission(auth.user, 'admin'):
```

Comment:
@samanehsan Actually, it may be better to modify `_get_children` so it only returns nodes that the current user has admin access to?